src/index.html

```diff
@@ -9,6 +9,63 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__wrap">
+        <div class="nav__logo">
+          <a href="#home" class="link__logo">
+            <img src="images/logo.png" alt="MOYO logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link first is-active">
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
```suggestion
            </li>
            
            <li class="nav__item">
```

Reply:
<img width="985" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/78ef5f27-9e10-4c07-9eb6-39adacb35b99">
Fix it in all places